Intersect.Client.Core/Interface/Menu/SelectCharacterWindow.cs

```diff
@@ -173,6 +173,14 @@ protected override void EnsureInitialized()
         SizeToChildren(recursive: true);
 
         LoadJsonUi(GameContentManager.UI.Menu, Graphics.Renderer?.GetResolutionString());
+
+        // in case the json load is changing the visibility of the arrows when it shouldn't
+        // lets ensure the right value
+        if (CharacterSelectionPreviews != default)
+        {
+            _selectCharacterRightButton.IsHidden = CharacterSelectionPreviews.Length <= 1;
+            _selectCharacterLeftButton.IsHidden = CharacterSelectionPreviews.Length <= 1;
+        }
```

Comment:
Move this all to `Show()` instead, otherwise they will display wrong if someone logs in, the server closes, the configuration is changed, and then they re-log in